ctapipe/tools/info.py

```diff
@@ -38,48 +40,87 @@
 )
 
 
-def main(args=None):
-    parser = get_parser(info)
-    parser.add_argument("--version", action="store_true", help="Print version number")
-    parser.add_argument(
-        "--tools", action="store_true", help="Print available command line tools"
-    )
-    parser.add_argument(
-        "--dependencies",
-        action="store_true",
-        help="Print available versions of dependencies",
-    )
-    parser.add_argument(
-        "--resources",
-        action="store_true",
-        help="Print available versions of dependencies",
-    )
-    parser.add_argument("--system", action="store_true", help="Print system info")
-    parser.add_argument(
-        "--all", dest="show_all", action="store_true", help="show all info"
-    )
-    parser.add_argument("--plugins", action="store_true", help="Print plugin info")
-    parser.add_argument(
-        "--datamodel", action="store_true", help="Print data model info"
+class InfoTool(Tool):
+    """
+    Print information about ctapipe and the current installation
+    """
+
+    name = "ctapipe-info"
+
+    tools = Bool(False, help="Print available command line tools").tag(config=True)
+    dependencies = Bool(False, help="Print available versions of dependencies").tag(
+        config=True
     )
-    parser.add_argument(
-        "--event-sources",
-        action="store_true",
-        help="Print available EventSource implementations",
+    resources = Bool(False, help="Print available versions of dependencies").tag(
+        config=True
     )
-    parser.add_argument(
-        "--reconstructors",
-        action="store_true",
-        help="Print available Reconstructor implementations",
+    system = Bool(False, help="Print system info").tag(config=True)
+    plugins = Bool(False, help="Print plugin info").tag(config=True)
+    datamodel = Bool(False, help="Print data model info").tag(config=True)
+    event_sources = Bool(False, help="Print available EventSource implementations").tag(
+        config=True
     )
-
-    args = parser.parse_args(args)
-
-    if len(sys.argv) <= 1:
-        parser.print_help()
-        sys.exit(1)
-
-    info(**vars(args))
+    reconstructors = Bool(
+        False, help="Print available Reconstructor implementations"
+    ).tag(config=True)
+
+    all = Bool(False, help="show all info").tag(config=True)
+
+    flags = {
+        "tools": ({"InfoTool": {"tools": True}}, "show available command-line tools"),
```

Comment:
Oddly, `ctapipe-info --tools` (or really any option other than `--help`) returns nothing, but e.g. `ctapipe info --tools` does what one expects.  Is there something broken with the `ctapipe-info` entry point maybe?